config/routes.rb

```diff
@@ -1,4 +1,4 @@
 Rails.application.routes.draw do
-  root to: "application#index"
+  mount_ember_app :exam_builder, to: "/", controller: "application", action: "index"
```

Comment:
Yes, this seems to be the crux of the issue. As I mentioned in thoughtbot/ember-cli-rails#292, `mount_ember_app` isn't an option for me, since we are including the Ember app on arbitrary Rails pages, like a widget. Is this no longer a supported use-case with Ember-CLI-Rails?